rules/dones-do-your-dones-include-a-url/rule.md

```diff
@@ -9,7 +9,9 @@ authors:
     url: https://ssw.com.au/people/cameron-shaw
   - title: Ulysses Maclaren
     url: https://ssw.com.au/people/ulysses-maclaren
-related: []
+related:
+  - add-context-reasoning-to-emails
+  - dones-do-you-include-useful-details-in-your-done-email
```

Comment:
Is this rule actually a duplicate of dones-do-you-include-useful-details-in-your-done-email ?

Reply:
Hi @jaydenalchin 

I see as:

- https://ssw.com.au/rules/add-context-reasoning-to-emails/
  Add context when you send an email

- https://ssw.com.au/rules/dones-do-you-include-useful-details-in-your-done-email/
  Add any useful info, such as context, screenshots, code changes etc... to "Done" emails 

- https://ssw.com.au/rules/dones-do-your-dones-include-a-url/
  Add URLs (links) to any email
  
  I think the 3 rules can live together... they are out there for a while... but if you disagree... I would recommend getting this PR through and opening a new Issue to look into merging them